tests/functional/health.ts

```diff
@@ -0,0 +1,67 @@
+/* tslint:disable:no-relative-imports */
+import { describe } from 'riteway'
+import { app } from '../../src/app'
+import { dbHelper } from '../helpers/database'
+import { getHealth } from '../helpers/health'
+import { delay, runtimeId } from '../helpers/utils'
+
+const PREFIX = `test-functional-node-poet-${runtimeId()}`
+const NODE_PORT = '28081'
+
+const blockchainSettings = {
+  MINIMUM_BLOCK_HEIGHT: 100,
+  ENABLE_TIMESTAMPING: true,
+  TIMESTAMP_INTERVAL_IN_SECONDS: 10,
+  BATCH_CREATION_INTERVAL_IN_SECONDS: 5,
+  READ_DIRECTORY_INTERVAL_IN_SECONDS: 5,
+}
+
+const createDatabase = async (prefix: string) => {
```

Comment:
Isn't this the same as `claim_with_data.ts`'s `createDatabase`? If so, can we move the function to a helper instead?